### PR TITLE
Mobile-first Portfolio Setup for review

### PR DESCRIPTION
**Pull request by amon-cofie**
This is my Mobile-first Portfolio Project. I've done the following things in it:

- Added a markup file (HTML)
- Added a stylesheet file (CSS)
- Added a .gitignore file
- Setup HTML,CSS linters
- Added a descriptive Readme file
- Added license for this project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,8 +63,262 @@ <h3>LET'S CONNECT</h3>
...
+              sign-ups required.
+            </p>
+            <ul class="lang-btns">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <div class="btn-container">
+              <button type="button">See Project</button>
+            </div>
```
Comment (smunozmo) [inline index.html:187]: - [ ] Your portfolio is looking great, congrats on it! 🎉 Let's make it even better by adding some states to your buttons as shown in the Figma template:

Screen Shot 2022-08-19 at 10 22 09 PM

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @amon-cofie, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Your styling is really good, excellent work! 😎 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (amon-cofie) [reply index.html:187]: Thank you very much @smunozmo

**Review by BenMukebo — APPROVED** [review]
### Hi @amon-cofie 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**